.circleci/config.yml

```diff
@@ -73,6 +73,12 @@ jobs:
           key: go-src-v1-{{ .Revision }}
           paths:
             - ".git"
+  proto:
+    executor: golang
```

Comment:
Waiting on @mircea-c to provide a correct image with all the deps we need and then we can merge.